pkg/kubecost/window.go

```diff
@@ -22,8 +22,8 @@ const (
 )
 
 var (
-	durationRegex       = regexp.MustCompile(`^(\d+)(m|h|d)$`)
-	durationOffsetRegex = regexp.MustCompile(`^(\d+)(m|h|d) offset (\d+)(m|h|d)$`)
+	durationRegex       = regexp.MustCompile(`^(\d+)(m|h|d|w)$`)
+	durationOffsetRegex = regexp.MustCompile(`^(\d+)(m|h|d|w) offset (\d+)(m|h|d)$`)
```

Comment:
If we're accepting w for durations, we should probably accept it for offsets, too.

Reply:
@Sean-Holcomb has some specific concerns related to allowing an offset measured in weeks.

Reply:
Discussed with Sean, we will go ahead and add it.